src/dotnet/commands/dotnet-new/CSharp_Console1.1/Program.cs

```diff
@@ -1,9 +1,12 @@
 using System;
 
-class Program
+namespace ConsoleApp
```

Comment:
Got it.  Thanks for catching up the guy who has been out for 2 months.  :)

One follow up question:  Won't hard-coding `ConsoleApp` as the namespace have the same issue as https://github.com/dotnet/sdk/issues/271?  Program.cs will have `namespace ConsoleApp` and the `New Class` item template will use `$safeprojectname$`, and you'll still need to add a using to Program.cs.